### ISSUE TITLE
(Bug report) Unable to uninstall

### ISSUE THREAD

**Issue by DarkZoomy** [TriliumNext/Trilium#3872]
### Trilium Version

0.59.4

### What operating system are you using?

Windows

### What is your setup?

Local (no sync)

### Operating System Version

Windows 11 Pro 22621.1555

### Description

Hello, 

i wasn't able to uninstall Trilium Notes. Neither over the uninstall menu nor over winget. It always will give me an error. I also can't find any uninstallment file in the Trilium folder so i had to delete all files manually. 

Windows: Windows 11 Pro 22H2 22621.1555
Winget: v1.4.10173
Chocolatey: 1.21

Greetings

**Comment by XXXJetfireXXX** [comment]
Yeah, Trilium has no automatic uninstaller. You just have to remove two folders in order to uninstall it: trilium-windows-x64, which contains the application itself, and trilium-data, which contains your database and its backups (in case you wanna delete that too)

**Comment by zadam** [comment]
@XXXJetfireXXX is correct, there's neither real installation process nor an uninstall process.

**Comment by eliandoran** [comment]
This should no longer be the case since we have an installer. Either way, Dosu's answer is correct.
